net-p2p/qbittorrent/qbittorrent-3.3.7.ebuild

```diff
@@ -74,11 +67,7 @@ src_configure() {
 		$(use_enable !X systemd) \
 		$(use_with !qt5 qt4)
 
-	if use qt5; then
-		eqmake5
-	else
-		eqmake4
-	fi
+	emake
```

Comment:
That doesn't answer my question. I don't care about what upstream says, our quality standards are higher. Removing the call to `eqmake[45]` will result in several QA issues.